app/javascript/src/components/Dashboard/Contacts/Table.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from "react";
+
+import { Table as FormikTable } from "neetoui";
```

Comment:
This module is imported from neetoui. Not formik.
```suggestion
import { Table as NeetoUITable } from "neetoui";
```